scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1009,6 +1008,13 @@ private void processClientHello(final ClientHello clientHello, final Record reco
 			LOGGER.debug(msg.toString());
 		}
 
+		// We do not support re-negociation as recommended in :
+		// https://tools.ietf.org/html/rfc7925#section-17
+		if (record.getEpoch() > 0) {
+			DTLSSession session = connection.getEstablishedSession();
+			send(new AlertMessage(AlertLevel.WARNING, AlertDescription.NO_RENEGOTIATION, clientHello.getPeer()),
+					session);
+		} else
```

Comment:
To limit the changes and comply with the source formatter:

```
   return;
}
```

instead of 

```
} else
```
?

Reply:
I can do that.
But just for information, I used the formatter and it was ok with that.